client/cody/src/services/CodyMenus.ts

```diff
@@ -84,23 +84,24 @@ export const AuthMenuOptions = {
 }
 
 export const LoginMenuOptionItems = [
+    {
+        id: 'enterprise',
+        label: 'Sign in to Sourcegraph Enterprise instance',
+        description: 'v5.1 and above',
+        totalSteps: 1,
+        picked: true,
+    },
     {
         id: 'token',
-        label: 'Sign in with URL and Access Token',
+        label: 'Sign in to Sourcegraph Enterprise instance via Access Token',
```

Comment:
I purposefully left of a general "URL and Access Token" option here — we know to use that for debugging and testing, but for all our users it's really just an Enterprise thing.

Reply:
I think there was a confusion with the previous sign in page where users thought that was only for Sourcegraph Enterprise instance when they can sign in to DotCom as well. Users on remote host that we do not support redirection is going to need to sign in using access token instead

Reply:
link to discord thread for ref: https://discord.com/channels/969688426372825169/1097918731260285008/1106495575421112352

Reply:
```suggestion
        label: 'Sign in to Sourcegraph Enterprise instance via Access Token',
```
Oh i was referring to this change. This is not limited to Sourcegraph Enterprise instance only.  Users on DotCom or Cody App can also use this option to login if they are on a platform that does not support redirection

Reply:
```suggestion
        label: 'Sign in to Sourcegraph Enterprise instance via Access Token',
```
Oh i was referring to this change. This is not limited to Sourcegraph Enterprise instance only.  Users on DotCom or Cody App can also use this option to login if they are on a platform that does not support redirection

Reply:
Oh right!  Maybe something like:

```ts
export const LoginMenuOptionItems = [
    {
        id: 'enterprise',
        label: 'Sign in to Sourcegraph Enterprise instance',
        description: 'v5.1 and above',
        totalSteps: 1,
        picked: true,
    },
    {
        id: 'token',
        label: 'Sign in to Sourcegraph Enterprise instance via Access Token',
        description: 'v5.0 and above',
        totalSteps: 2,
    },
    {
        id: 'dotcom',
        label: 'Sign in to Sourcegraph.com',
        totalSteps: 0,
    },
    {
        id: 'dotcom-token',
        label: 'Sign in to Sourcegraph.com via Access Token',
        totalSteps: 1,
    },
    {
        id: 'dotcom-token',
        label: 'Sign in with URL and Access Token',
        totalSteps: 2,
    },
]
```

but we split it with some separator labels, "enterprise", "sourcegraph.com", "other"

This is getting far from simple… but I think at least it's logical and easy to pick the right one for the most common cases.

Or could we do without the `"Sign in to Sourcegraph.com via Access Token"` option in this list?